### PR TITLE
create deploy/ansible/ directory with stellar-core setup

### PR DESCRIPTION
**Pull request by oryband**
- initially this directory includes playbooks and roles to deploy:
  stellar-core + horizon for multiple network (passphrases)
  and on multiple regions.
- sensitive configuration is saved on AWS SSM Parameter Store.
  see specific roles that handle getting/putting information from/to there
- will update this directory with additional host inventory in future commits
- also add a testing setup using docker for deploying playbooks locally. don't freak out from the ssh key committed here - it's for ansible to ssh and deploy into the docker container for testing
  this speeds up development a lot. see deploy/image/ directory
- also note the `timesyncd` role which replaces `ntp` from here on, and uses AWS default NTP address which doesn't require any security role to set up

### CODE REVIEW

File: deploy/ansible/playbooks/roles/horizon/templates/docker-compose.yml.j2 [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    environment:
+      DATABASE_URL: "postgres://{{this_node.db.user}}:{{this_node.db.password}}@{{this_node.db.host}}/{{this_node.db.name}}?sslmode={{this_node.db.ssl_mode|default('disable')}}"
+      STELLAR_CORE_DATABASE_URL: "postgres://{{this_node.stellar_core.db.user}}:{{this_node.stellar_core.db.password}}@{{this_node.stellar_core.db.host}}/{{this_node.stellar_core.db.name}}?sslmode={{this_node.stellar_core.db.ssl_mode|default('disable')}}"
+      STELLAR_CORE_URL: "{{this_node.stellar_core.url}}"
+      LOG_LEVEL: "{{horizon.log_level|default('warn')}}"
+      INGEST: "true"
+      PER_HOUR_RATE_LIMIT: "999999"
+      NETWORK_PASSPHRASE: "{{stellar_core.network_passphrase}}"
+      FRIENDBOT_URL: '{{horizon.friendbot_url|default("")}}'
+
```
Comment (Ronserruya) [inline deploy/ansible/playbooks/roles/horizon/templates/docker-compose.yml.j2]: what about HISTORY_RETENTION_COUNT?

Comment (oryband) [reply deploy/ansible/playbooks/roles/horizon/templates/docker-compose.yml.j2]: Thanks, forgot that one

**Comment by oryband** [comment]
> I went over this swiftly. Overall looks good. You made smaller role modules this time, no?

@doodyparizada yes, easier to re-use and serve as a guide for others.

**Comment by oryband** [comment]
@gadisr please review as well

**Pull request merged**